addons/im_livechat/models/mail_channel.py

```diff
@@ -143,11 +143,7 @@ def _send_history_message(self, pid, page_history):
         if page_history:
             html_links = ['<li><a href="%s" target="_blank">%s</a></li>' % (page, page) for page in page_history]
             message_body = '<span class="o_mail_notification"><ul>%s</ul></span>' % (''.join(html_links))
-        self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', pid), {
-            'body': message_body,
-            'channel_ids': self.ids,
-            'info': 'transient_message',
-        })
+        self._send_transient_message(self.env['res.partner'].browse(pid), message_body)
```

Comment:
It looks like `<span class="o_mail_notification">` is already part of `_send_transient_message`, so it would add it in double with current change.

Reply:
Damn I forgot to remove it. It's done now !